### PR TITLE
Delete object bug

### PR DESCRIPTION
**Pull request by maggarwal13**

### CODE REVIEW

File: whois-update/src/main/java/net/ripe/db/whois/update/handler/SingleUpdateHandler.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ public void handle(final Origin origin, final Keyword keyword, final Update upda
         }
 
         if (action == Action.DELETE && originalObject == null) {
-            updateContext.addMessage(update, UpdateMessages.objectNotFound(updatedObject.getFormattedKey()));
+            updateContext.addMessage(update, UpdateMessages.objectNotFound(update.getSubmittedObject().getFormattedKey()));
```
Comment (eshryane) [inline whois-update/src/main/java/net/ripe/db/whois/update/handler/SingleUpdateHandler.java:105]: So the error message will contain the key as the user submitted it?

File: whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java [hunk 2]
```diff
@@ -4707,6 +4707,33 @@ public void delete_route_out_of_region_redirect() {
...
+        databaseHelper.addObject(
+                "domain:        193.in-addr.arpa4\n" +
+                        "nserver:         test.ns.\n" +
+                        "mnt-by:          OWNER-MNT\n" +
+                        "source:          TEST\n");
+
+        final WhoisResources whoisResources = RestTest.target(getPort(), "whois/test/domain/193.in-addr.arpa4")
+                .request()
+                .get(WhoisResources.class);
+        assertThat(whoisResources.getWhoisObjects().get(0).getPrimaryKey().get(0).getValue(), is("193.in-addr.arpa4"));
```
Comment (eshryane) [inline whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java:4721]: Do you need to test the primary key in the test? It seems superfluous

File: whois-update/src/main/java/net/ripe/db/whois/update/handler/SingleUpdateHandler.java (hunk 1 above)
Comment (maggarwal13) [reply whois-update/src/main/java/net/ripe/db/whois/update/handler/SingleUpdateHandler.java:105]: Yes, like it was before

File: whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java (hunk 2 above)
Comment (maggarwal13) [reply whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java:4721]: Its just a way to test that before delete key exists and after it does not. I can remove that if you prefer that

Comment (eshryane) [reply whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java:4721]: I don't think it's needed, and can be safely removed

File: whois-update/src/main/java/net/ripe/db/whois/update/handler/SingleUpdateHandler.java (hunk 1 above)
Comment (eshryane) [reply whois-update/src/main/java/net/ripe/db/whois/update/handler/SingleUpdateHandler.java:105]: Great, thanks

File: whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java (hunk 2 above)
Comment (maggarwal13) [reply whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java]: Done

**Review by eshryane — APPROVED** [review]

**Pull request merged**